docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -56,6 +56,11 @@ On the **Dashboards** page, you can browse and manage folders and dashboards. Th
 As of Grafana 10.2, there is no longer a special **General** folder. Dashboards without a folder are now shown at the top level alongside folders.
 {{% /admonition %}}
 
+### Shared with me
+
+The **Shared with me** section displays folders and dashboards that are directly shared with a user which would otherwise not be visible in the standard dashboards tree.
```

Comment:
Changing this to second person vs third

```suggestion
The **Shared with me** section displays folders and dashboards that are directly shared with you that wouldn't otherwise be visible in the standard dashboards tree.
```

Reply:
Also, is there another word we can use instead of "tree"--it's just sort of been introduced here without any previous definition of what we mean. Does the word "list" suffice?